contentcuration/contentcuration/frontend/channelEdit/components/AddRelatedResourcesModal.vue

```diff
@@ -3,15 +3,18 @@
   <VDialog
     :value="true"
     fullscreen
+    app
     hide-overlay
     transition="dialog-bottom-transition"
     lazy
     scrollable
   >
-    <VCard>
+    <VCard :style="{'height': '100%'}">
       <VToolbar
         dark
+        app
```

Comment:
Just need to double check, when you leave this view, do the other layouts shift?

Reply:
I've just checked it and haven't noticed any shifts